src/index.html

```diff
@@ -9,6 +9,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="header__a"><img src="images/logo.png" alt="moyo-logo" ></a>
+      <nav>
+        <ul class="nav__list">
+          <li><a href="#apple" class="is-active nav__a">Apple</a></li>
```

Comment:
check your code style, it would be more readable
also, add an empty blank between siblings
```suggestion
        <ul class="nav__list">
          <li>
            <a href="#apple" class="is-active nav__a">
              Apple
            </a>
          </li>
          
```